guides/advanced_guides/known_limitations.md

```diff
@@ -0,0 +1,31 @@
+# Known limitations
+
+MeiliSearch has known limitations, some of which may be addressed in future versions.
+
+Some of these limitations have been made by MeiliSearch developers for relevacy and/or performance while others limitations are enforced by LMDB, the key-value store used by MeiliSearch.
+
+## Design limitations
+
+### Database size
+
+MeiliSearch uses two databases: the first one for storage and the second one for updates.
+On launch LMDB needs to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory.
+
+The maxime database size is by default __100GiB__ for each databases. This size can me modified using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration guide](/guides/advanced_guides/configuration.md#max-mdb-size).
+
+### Number of indexes
+
+You can create __up to 200 indexes__ in MeiliSearch. This limit has been hard set for performance reasons.
+
+### Maximum words per attribute
+
+A maximum of __1000 words par attribute__ can be indexed. This means that if an attribute contains more than 1000 words, only the first 1000 words will be indexed and the rest will be silently ignored.
+
+This limit is enforced for relevancy reasons. The more words there are in a given attribute, the less relevant the search queries will be.
```

Comment:
(change if you agree if the previous suggestion)
```suggestion
This limit is enforced for relevancy reasons. The more values are linked to a given attribute, the less relevant the search queries will be.
```

Reply:
Per my answer on the previous suggestion I disagree with this suggestion